src/reducers/__tests__/query.spec.jsx

```diff
@@ -584,24 +627,52 @@ describe( 'reducer:query', () => {
 
       it( "adds all filters if they didn't exist", () => {
         expect( target( {}, action ) ).toEqual( {
-          enablePer1000: false,
```

Comment:
🤔

Reply:
There's a separate test for this when it's in Map mode.

Reply:
I see how this is working now.  I went to the file itself.  The diffs weren't making sense